src/components/backend-ai-resource-policy-list.ts

```diff
@@ -773,27 +780,24 @@ export default class BackendAIResourcePolicyList extends BackendAIPage {
       textfield.value = Math.round(textfield.value);
     }
 
-    if (textfield.value <= 0) {
-      // concurrency job and container-per-session limit must be upper than 0.
-      textfield.value = ((textfield.id === 'concurrency-limit') || (textfield.id === 'container-per-session-limit')) ? 1 : 0;
-    }
-
-    if (!textfield.valid) {
+    if (!textfield.isUiValid) {
       const decimal_point: number = (textfield.step) ? countDecimals(textfield.step) : 0;
       if (decimal_point > 0) {
         textfield.value = Math.min(textfield.value, (textfield.value < 0) ? textfield.min : textfield.max).toFixed(decimal_point);
       } else {
         textfield.value = Math.min(Math.round(textfield.value), (textfield.value < 0) ? textfield.min : textfield.max);
       }
     }
-    // automatically check when textfield is min
+
+    // automatically check when textfield is min or max
     if (checkbox) {
-      textfield.disabled = checkbox.checked = (textfield.value == parseFloat(textfield.min));
+      textfield.disabled = checkbox.checked = ((textfield.value == parseFloat(textfield.min)) || (textfield.value == parseFloat(textfield.max)));
```

Comment:
How about `(textfield.value <= parseFloat(textfield.min)) || (textfield.value >= parseFloat(textfield.max))`?

Reply:
Your right. It's better to cover values under min and max.